final_task/README.md

```diff
@@ -1,3 +1,42 @@
-# Your readme here
-Some text.
-Checkout how to write this file using *markdown*.
+# RSS reader
+Python command line RSS-reader.
+
+## Usage
+
+```shell
+$ rss_reader [-h] [--version] [--json] [--verbose] [--limit LIMIT] source
+
+Pure python command-line RSS reader
+
+positional arguments:
+  source         RSS URL
+
+optional arguments:
+  -h, --help     show this help message and exit
+  --version      Print version info
+  --json         Print result as JSON in stdout
+  --verbose      Outputs verbose status messages
+  --limit LIMIT  Limit news topics if this parameter provided
+
+```
+
+## Installation
+* For installation you need setuptools
+
+```shell
+python -m pip install source_name
+```
+
+
+## JSON structure
+```shell
+{
+    "news_outlett_name": "news outlett name",
+    "news_title": "news title",
+    "pub_date": "publication date, Sun, 17 Nov 2019 10:44:20 -0500",
+    "news_link": "news link",
+    "news_description": "news description",
+    "img_alt": "image alternative if exist",
+    "img_src": "image link if exist",
+}
+```
```

Comment:
Please add information how to launch your app using setuptools

Reply:
I actually have a problem with it. To launch from source you can simply type "rss_reader.py [-h] [--version] [--json] [--verbose] [--limit LIMIT] source". But for installation via setuptools I can't name entry_point console_script from setup.py the same name as my entry point rss_reader.py. With other names it works fine. For now it works like "rss_reader [-h] [--version] [--json] [--verbose] [--limit LIMIT] source". I suppose it isn't OK so can you give me any tips, please. Or should I leave it be and just add the information about how to launch the app using setuptools?

Reply:
Try to bypass this problem using `sys.path.append` if you have troubles when import does not work either for setuptools or launching from source